src/ray/core_worker/core_worker.cc

```diff
@@ -1176,10 +1176,6 @@ void CoreWorker::Exit(
   }
 
   RAY_LOG(DEBUG) << "Exit signal received, remove all local references.";
```

Comment:
Should this log be also moved inside the `not_actor_task` branch?

Reply:
yes it should! let me fix

Reply:
fixed, thx